pkg/ccl/telemetryccl/telemetryccl_test: TestTelemetry failed

pkg/ccl/telemetryccl/telemetryccl_test.TestTelemetry [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5582830?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5582830?buildTab=artifacts#/) on master @ [096ed43069fa5ce48e2fd276b9cc0342aaf7d223](https://github.com/cockroachdb/cockroach/commits/096ed43069fa5ce48e2fd276b9cc0342aaf7d223):


```
        ----
        sql.multiregion.zone_configuration.override.user
    telemetry.go:111: 
        testdata/telemetry/multiregion:450:
        feature-usage [0 args]
        ALTER TABLE t9 CONFIGURE ZONE USING gc.ttlseconds=5
        ----
    telemetry.go:111: 
        testdata/telemetry/multiregion:454:
        feature-usage [0 args]
        SET override_multi_region_zone_config = true;
        ALTER TABLE t9 SET LOCALITY GLOBAL;
        SET override_multi_region_zone_config = false
        ----
        sql.multiregion.zone_configuration.override.system.table
    telemetry.go:111: 
        testdata/telemetry/multiregion:462:
        feature-usage [0 args]
        ALTER TABLE t9 SET LOCALITY REGIONAL BY TABLE IN PRIMARY REGION;
        ----
    telemetry.go:111: 
        testdata/telemetry/multiregion:469:
        feature-usage [0 args]
        SET override_multi_region_zone_config = true;
        ALTER TABLE t9 SET LOCALITY GLOBAL;
        SET override_multi_region_zone_config = false
        ----
        sql.multiregion.zone_configuration.override.system.table
    telemetry.go:111: 
        testdata/telemetry/multiregion:476:
        feature-usage [0 args]
        SET override_multi_region_zone_config = true;
        ALTER DATABASE d CONFIGURE ZONE USING num_replicas=10;
        SET override_multi_region_zone_config = false
        ----
        sql.multiregion.zone_configuration.override.user
    telemetry.go:111: 
        testdata/telemetry/multiregion:483:
        feature-usage [0 args]
        ALTER DATABASE d CONFIGURE ZONE USING gc.ttlseconds=5;
        ----
    telemetry.go:111: 
        testdata/telemetry/multiregion:487:
        feature-usage [0 args]
        SET override_multi_region_zone_config = true;
        ALTER DATABASE d ADD REGION "us-east-1";
        SET override_multi_region_zone_config = false
        ----
        sql.multiregion.zone_configuration.override.system.database
        --- PASS: TestTelemetry/multiregion/server (156.92s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTelemetry.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-17102

pkg/ccl/telemetryccl/telemetryccl_test.TestTelemetry [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7686316?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7686316?buildTab=artifacts#/) on master @ [b5be006bedd7d3cedc3fb3d2248df168e3d64be2](https://github.com/cockroachdb/cockroach/commits/b5be006bedd7d3cedc3fb3d2248df168e3d64be2):


```
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:229:
        feature-usage [0 args]
        ALTER TABLE t4 SET LOCALITY REGIONAL BY ROW
        ----
        sql.multiregion.alter_table.locality.from.global.to.regional_by_row
    telemetry.go:110: 
        testdata/telemetry/multiregion:234:
        exec [0 args]
        ALTER TABLE t4 SET LOCALITY GLOBAL
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:238:
        feature-usage [0 args]
        ALTER TABLE t4 SET LOCALITY GLOBAL
        ----
        sql.multiregion.alter_table.locality.from.global.to.global
    telemetry.go:110: 
        testdata/telemetry/multiregion:243:
        exec [0 args]
        ALTER TABLE t4 SET LOCALITY GLOBAL
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:247:
        feature-usage [0 args]
        ALTER TABLE t4 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
        sql.multiregion.alter_table.locality.from.global.to.regional_by_table_in
    telemetry.go:110: 
        testdata/telemetry/multiregion:252:
        exec [0 args]
        ALTER TABLE t4 SET LOCALITY GLOBAL;
        ALTER TABLE t4 ADD COLUMN cr crdb_internal_region NOT NULL
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:257:
        feature-usage [0 args]
        ALTER TABLE t4 SET LOCALITY REGIONAL BY ROW AS "cr"
        ----
        sql.multiregion.alter_table.locality.from.global.to.regional_by_row_as
    telemetry.go:110: 
        testdata/telemetry/multiregion:262: ALTER TABLE t4 SET LOCALITY GLOBAL
        expected:
        
        found:
        error: pq: protected timestamp record does not exist
        --- FAIL: TestTelemetry/multiregion/server (43.45s)
=== RUN   TestTelemetry/multiregion
    --- FAIL: TestTelemetry/multiregion (59.35s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTelemetry.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/telemetryccl/telemetryccl_test.TestTelemetry [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7721520?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7721520?buildTab=artifacts#/) on master @ [1a6e9f885baa124d5ff2996adb966ea15a1a9b2b](https://github.com/cockroachdb/cockroach/commits/1a6e9f885baa124d5ff2996adb966ea15a1a9b2b):


```
        testdata/telemetry/multiregion:306:
        exec [0 args]
        ALTER TABLE t5 ADD COLUMN cr crdb_internal_region NOT NULL
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:310:
        feature-usage [0 args]
        ALTER TABLE t5 SET LOCALITY REGIONAL BY ROW AS "cr"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_row.to.regional_by_row_as
    telemetry.go:110: 
        testdata/telemetry/multiregion:315:
        exec [0 args]
        ALTER TABLE t5 SET LOCALITY REGIONAL BY ROW
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:319:
        feature-usage [0 args]
        ALTER TABLE t5 SET LOCALITY REGIONAL BY TABLE
        ----
        sql.multiregion.alter_table.locality.from.regional_by_row.to.regional_by_table
    telemetry.go:110: 
        testdata/telemetry/multiregion:324:
        exec [0 args]
        ALTER TABLE t5 SET LOCALITY REGIONAL BY ROW
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:332:
        feature-usage [0 args]
        ALTER TABLE t6 SET LOCALITY REGIONAL BY ROW
        ----
        sql.multiregion.alter_table.locality.from.regional_by_row_as.to.regional_by_row
    telemetry.go:110: 
        testdata/telemetry/multiregion:337:
        exec [0 args]
        ALTER TABLE t6 SET LOCALITY REGIONAL BY ROW AS "cr"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:341:
        feature-usage [0 args]
        ALTER TABLE t6 SET LOCALITY GLOBAL
        ----
        sql.multiregion.alter_table.locality.from.regional_by_row_as.to.global
    telemetry.go:110: 
        testdata/telemetry/multiregion:346: ALTER TABLE t6 SET LOCALITY REGIONAL BY ROW AS "cr"
        expected:
        
        found:
        error: pq: protected timestamp record does not exist
        --- FAIL: TestTelemetry/multiregion/server (85.23s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTelemetry.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/telemetryccl/telemetryccl_test.TestTelemetry [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7743737?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7743737?buildTab=artifacts#/) on master @ [2b595c5214aead3e7d4f47f9a1dedce57dba6e7f](https://github.com/cockroachdb/cockroach/commits/2b595c5214aead3e7d4f47f9a1dedce57dba6e7f):


```
        testdata/telemetry/multiregion:337:
        exec [0 args]
        ALTER TABLE t6 SET LOCALITY REGIONAL BY ROW AS "cr"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:341:
        feature-usage [0 args]
        ALTER TABLE t6 SET LOCALITY GLOBAL
        ----
        sql.multiregion.alter_table.locality.from.regional_by_row_as.to.global
    telemetry.go:110: 
        testdata/telemetry/multiregion:346:
        exec [0 args]
        ALTER TABLE t6 SET LOCALITY REGIONAL BY ROW AS "cr"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:350:
        feature-usage [0 args]
        ALTER TABLE t6 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_row_as.to.regional_by_table_in
    telemetry.go:110: 
        testdata/telemetry/multiregion:355:
        exec [0 args]
        ALTER TABLE t6 SET LOCALITY REGIONAL BY ROW AS "cr"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:359:
        feature-usage [0 args]
        ALTER TABLE t6 SET LOCALITY REGIONAL BY ROW AS "cr"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_row_as.to.regional_by_row_as
    telemetry.go:110: 
        testdata/telemetry/multiregion:364:
        exec [0 args]
        ALTER TABLE t6 SET LOCALITY REGIONAL BY ROW AS "cr"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:368:
        feature-usage [0 args]
        ALTER TABLE t6 SET LOCALITY REGIONAL BY TABLE
        ----
        sql.multiregion.alter_table.locality.from.regional_by_row_as.to.regional_by_table
    telemetry.go:110: 
        testdata/telemetry/multiregion:373: ALTER TABLE t6 SET LOCALITY REGIONAL BY ROW AS "cr"
        expected:
        
        found:
        error: pq: protected timestamp record does not exist
        --- FAIL: TestTelemetry/multiregion/server (100.63s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTelemetry.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/telemetryccl/telemetryccl_test.TestTelemetry [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9639391?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9639391?buildTab=artifacts#/) on master @ [3382b2737ceb72de9b14d1d5923773718be5bca7](https://github.com/cockroachdb/cockroach/commits/3382b2737ceb72de9b14d1d5923773718be5bca7):


```
    telemetry.go:110: 
        testdata/telemetry/multiregion:166:
        feature-usage [0 args]
        ALTER TABLE t1 SET LOCALITY REGIONAL BY TABLE
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table.to.regional_by_table
    telemetry.go:110: 
        testdata/telemetry/multiregion:171:
        exec [0 args]
        ALTER TABLE t1 SET LOCALITY REGIONAL BY TABLE
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:179:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY ROW
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_row
    telemetry.go:110: 
        testdata/telemetry/multiregion:184:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:188:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY GLOBAL
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.global
    telemetry.go:110: 
        testdata/telemetry/multiregion:193:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:197:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_table_in
    telemetry.go:110: 
        testdata/telemetry/multiregion:202:
         ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2";
        ALTER TABLE t3 ADD COLUMN cr crdb_internal_region NOT NULL
        expected:
        
        found:
        error: pq: replica unavailable: (n1,s1):1 unable to serve request to r32:/NamespaceTable/{30-Max} [(n1,s1):1, (n3,s3):2, (n2,s2):3, next=4, gen=4]: closed timestamp: 1681714713.479739306,0 (2023-04-17 06:58:33); raft status: {"id":"1","term":6,"vote":"1","commit":230,"lead":"1","raftState":"StateLeader","applied":230,"progress":{"1":{"match":280,"next":281,"state":"StateReplicate"},"2":{"match":230,"next":283,"state":"StateReplicate"},"3":{"match":229,"next":283,"state":"StateReplicate"}},"leadtransferee":"0"}: operation "probe" timed out after 1m0.169s (given timeout 1m0s): result is ambiguous: after 60.16s of attempting command: context deadline exceeded
        --- FAIL: TestTelemetry/multiregion/server (550.22s)
=== RUN   TestTelemetry/multiregion
    --- FAIL: TestTelemetry/multiregion (598.01s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTelemetry.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/telemetryccl/telemetryccl_test.TestTelemetry [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9659664?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9659664?buildTab=artifacts#/) on master @ [1cd507a7c6582fd91bb56123e62bd4fde45c4d22](https://github.com/cockroachdb/cockroach/commits/1cd507a7c6582fd91bb56123e62bd4fde45c4d22):


```
        exec [0 args]
        ALTER TABLE t4 SET LOCALITY GLOBAL
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:266:
        feature-usage [0 args]
        ALTER TABLE t4 SET LOCALITY REGIONAL BY TABLE
        ----
        sql.multiregion.alter_table.locality.from.global.to.regional_by_table
    telemetry.go:110: 
        testdata/telemetry/multiregion:271:
        exec [0 args]
        ALTER TABLE t4 SET LOCALITY GLOBAL
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:279:
        feature-usage [0 args]
        ALTER TABLE t5 SET LOCALITY REGIONAL BY ROW
        ----
        sql.multiregion.alter_table.locality.from.regional_by_row.to.regional_by_row
    telemetry.go:110: 
        testdata/telemetry/multiregion:284:
        exec [0 args]
        ALTER TABLE t5 SET LOCALITY REGIONAL BY ROW
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:288:
        feature-usage [0 args]
        ALTER TABLE t5 SET LOCALITY GLOBAL
        ----
        sql.multiregion.alter_table.locality.from.regional_by_row.to.global
    telemetry.go:110: 
        testdata/telemetry/multiregion:293:
        exec [0 args]
        ALTER TABLE t5 SET LOCALITY REGIONAL BY ROW
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:297:
        feature-usage [0 args]
        ALTER TABLE t5 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_row.to.regional_by_table_in
    telemetry.go:110: 
        testdata/telemetry/multiregion:302:
         ALTER TABLE t5 SET LOCALITY REGIONAL BY ROW;
        expected:
        
        found:
        error: pq: polling for queued jobs to complete: poll-show-jobs: replica unavailable: (n1,s1):1 unable to serve request to r17:/Table/1{5-6} [(n1,s1):1, (n2,s2):2, (n3,s3):3, next=4, gen=4]: closed timestamp: 1681804302.987147573,0 (2023-04-18 07:51:42); raft status: {"id":"1","term":8,"vote":"1","commit":2971,"lead":"1","raftState":"StateLeader","applied":2971,"progress":{"2":{"match":2965,"next":2998,"state":"StateReplicate"},"3":{"match":2971,"next":2998,"state":"StateReplicate"},"1":{"match":2995,"next":2996,"state":"StateReplicate"}},"leadtransferee":"0"}: have been waiting 61.00s for slow proposal RequestLease [/Table/15,/Min)
        --- FAIL: TestTelemetry/multiregion/server (906.97s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTelemetry.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/telemetryccl/telemetryccl_test.TestTelemetry [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9681419?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9681419?buildTab=artifacts#/) on master @ [a4ab6b6243f509e62993b4d81855f23470d9901a](https://github.com/cockroachdb/cockroach/commits/a4ab6b6243f509e62993b4d81855f23470d9901a):


```
    telemetry.go:110: 
        testdata/telemetry/multiregion:179:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY ROW
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_row
    telemetry.go:110: 
        testdata/telemetry/multiregion:184:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:188:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY GLOBAL
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.global
    telemetry.go:110: 
        testdata/telemetry/multiregion:193:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:197:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_table_in
    telemetry.go:110: 
        testdata/telemetry/multiregion:202:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2";
        ALTER TABLE t3 ADD COLUMN cr crdb_internal_region NOT NULL
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:207:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY ROW AS "cr"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_row_as
    telemetry.go:110: 
        testdata/telemetry/multiregion:212:
         ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        expected:
        
        found:
        error: pq: polling for queued jobs to complete: poll-show-jobs: replica unavailable: (n1,s1):1 unable to serve request to r17:/Table/1{5-6} [(n1,s1):1, (n3,s3):2, (n2,s2):3, next=4, gen=4]: closed timestamp: 1681889481.412858099,0 (2023-04-19 07:31:21); raft status: {"id":"1","term":6,"vote":"1","commit":1898,"lead":"1","raftState":"StateLeader","applied":1898,"progress":{"1":{"match":1921,"next":1922,"state":"StateReplicate"},"2":{"match":1898,"next":1923,"state":"StateReplicate"},"3":{"match":1894,"next":1923,"state":"StateReplicate"}},"leadtransferee":"0"}: have been waiting 62.50s for slow proposal RequestLease [/Table/15,/Min)
        --- FAIL: TestTelemetry/multiregion/server (622.50s)
=== RUN   TestTelemetry/multiregion
    --- FAIL: TestTelemetry/multiregion (661.87s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTelemetry.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/telemetryccl/telemetryccl_test.TestTelemetry [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9700649?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9700649?buildTab=artifacts#/) on master @ [b767225a6893ea6e81d50618d634dbba92d3b680](https://github.com/cockroachdb/cockroach/commits/b767225a6893ea6e81d50618d634dbba92d3b680):


```
        ALTER TABLE t1 SET LOCALITY REGIONAL BY TABLE
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:179:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY ROW
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_row
    telemetry.go:110: 
        testdata/telemetry/multiregion:184:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:188:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY GLOBAL
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.global
    telemetry.go:110: 
        testdata/telemetry/multiregion:193:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:197:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_table_in
    telemetry.go:110: 
        testdata/telemetry/multiregion:202:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2";
        ALTER TABLE t3 ADD COLUMN cr crdb_internal_region NOT NULL
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:207:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY ROW AS "cr"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_row_as
    telemetry.go:110: 
        testdata/telemetry/multiregion:212:
         ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        expected:
        
        found:
        error: pq: polling for queued jobs to complete: poll-show-jobs: replica unavailable: (n2,s2):2 unable to serve request to r17:/Table/1{5-6} [(n1,s1):1, (n2,s2):2, (n3,s3):3, next=4, gen=4]: closed timestamp: 1681975989.764003767,0 (2023-04-20 07:33:09); raft status: {"id":"2","term":9,"vote":"2","commit":1875,"lead":"2","raftState":"StateLeader","applied":1875,"progress":{"1":{"match":1857,"next":1899,"state":"StateReplicate"},"2":{"match":1897,"next":1898,"state":"StateReplicate"},"3":{"match":1875,"next":1899,"state":"StateReplicate"}},"leadtransferee":"0"}: have been waiting 60.50s for slow proposal RequestLease [/Table/15,/Min)
        --- FAIL: TestTelemetry/multiregion/server (686.24s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTelemetry.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/telemetryccl/telemetryccl_test.TestTelemetry [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9721907?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9721907?buildTab=artifacts#/) on master @ [8124bff03d073f35f4d2b6a2048c7f4417d757d9](https://github.com/cockroachdb/cockroach/commits/8124bff03d073f35f4d2b6a2048c7f4417d757d9):


```
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:197:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_table_in
    telemetry.go:110: 
        testdata/telemetry/multiregion:202:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2";
        ALTER TABLE t3 ADD COLUMN cr crdb_internal_region NOT NULL
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:207:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY ROW AS "cr"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_row_as
    telemetry.go:110: 
        testdata/telemetry/multiregion:212:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:216:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_table
    telemetry.go:110: 
        testdata/telemetry/multiregion:221:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:229:
        feature-usage [0 args]
        ALTER TABLE t4 SET LOCALITY REGIONAL BY ROW
        ----
        sql.multiregion.alter_table.locality.from.global.to.regional_by_row
    telemetry.go:110: 
        testdata/telemetry/multiregion:234:
         ALTER TABLE t4 SET LOCALITY GLOBAL
        expected:
        
        found:
        error: pq: polling for queued jobs to complete: poll-show-jobs: replica unavailable: (n1,s1):1 unable to serve request to r26:/Table/2{4-5} [(n1,s1):1, (n2,s2):2, (n3,s3):3, next=4, gen=4]: closed timestamp: 1682059141.173562803,0 (2023-04-21 06:39:01); raft status: {"id":"1","term":22,"vote":"1","commit":402,"lead":"0","raftState":"StateCandidate","applied":402,"progress":{},"leadtransferee":"0"}: have been waiting 63.00s for slow proposal RequestLease [/Table/24,/Min)
        --- FAIL: TestTelemetry/multiregion/server (1174.61s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTelemetry.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/telemetryccl/telemetryccl_test.TestTelemetry [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9739260?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9739260?buildTab=artifacts#/) on master @ [10665f9856d9c7ca51557bdf5b48fe9df796228b](https://github.com/cockroachdb/cockroach/commits/10665f9856d9c7ca51557bdf5b48fe9df796228b):


```
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:229:
        feature-usage [0 args]
        ALTER TABLE t4 SET LOCALITY REGIONAL BY ROW
        ----
        sql.multiregion.alter_table.locality.from.global.to.regional_by_row
    telemetry.go:110: 
        testdata/telemetry/multiregion:234:
        exec [0 args]
        ALTER TABLE t4 SET LOCALITY GLOBAL
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:238:
        feature-usage [0 args]
        ALTER TABLE t4 SET LOCALITY GLOBAL
        ----
        sql.multiregion.alter_table.locality.from.global.to.global
    telemetry.go:110: 
        testdata/telemetry/multiregion:243:
        exec [0 args]
        ALTER TABLE t4 SET LOCALITY GLOBAL
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:247:
        feature-usage [0 args]
        ALTER TABLE t4 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
        sql.multiregion.alter_table.locality.from.global.to.regional_by_table_in
    telemetry.go:110: 
        testdata/telemetry/multiregion:252:
        exec [0 args]
        ALTER TABLE t4 SET LOCALITY GLOBAL;
        ALTER TABLE t4 ADD COLUMN cr crdb_internal_region NOT NULL
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:257:
        feature-usage [0 args]
        ALTER TABLE t4 SET LOCALITY REGIONAL BY ROW AS "cr"
        ----
        sql.multiregion.alter_table.locality.from.global.to.regional_by_row_as
    telemetry.go:110: 
        testdata/telemetry/multiregion:262:
         ALTER TABLE t4 SET LOCALITY GLOBAL
        expected:
        
        found:
        error: pq: polling for queued jobs to complete: poll-show-jobs: replica unavailable: (n1,s1):1 unable to serve request to r17:/Table/1{5-6} [(n1,s1):1, (n3,s3):2, (n2,s2):3, next=4, gen=4]: closed timestamp: 1682149681.913398940,0 (2023-04-22 07:48:01); raft status: {"id":"1","term":10,"vote":"1","commit":2518,"lead":"1","raftState":"StateLeader","applied":2516,"progress":{"3":{"match":2517,"next":2539,"state":"StateReplicate"},"1":{"match":2538,"next":2539,"state":"StateReplicate"},"2":{"match":2518,"next":2539,"state":"StateReplicate"}},"leadtransferee":"0"}: have been waiting 61.50s for slow proposal RequestLease [/Table/15,/Min)
        --- FAIL: TestTelemetry/multiregion/server (788.92s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTelemetry.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/telemetryccl/telemetryccl_test.TestTelemetry [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9751575?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9751575?buildTab=artifacts#/) on master @ [23ea65e4b5cfaa5e02298cddfbe8f9fbe581456b](https://github.com/cockroachdb/cockroach/commits/23ea65e4b5cfaa5e02298cddfbe8f9fbe581456b):


```
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:197:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_table_in
    telemetry.go:110: 
        testdata/telemetry/multiregion:202:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2";
        ALTER TABLE t3 ADD COLUMN cr crdb_internal_region NOT NULL
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:207:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY ROW AS "cr"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_row_as
    telemetry.go:110: 
        testdata/telemetry/multiregion:212:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:216:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_table
    telemetry.go:110: 
        testdata/telemetry/multiregion:221:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:229:
        feature-usage [0 args]
        ALTER TABLE t4 SET LOCALITY REGIONAL BY ROW
        ----
        sql.multiregion.alter_table.locality.from.global.to.regional_by_row
    telemetry.go:110: 
        testdata/telemetry/multiregion:234:
         ALTER TABLE t4 SET LOCALITY GLOBAL
        expected:
        
        found:
        error: pq: polling for queued jobs to complete: poll-show-jobs: deleting ambiguously created lease: operation "intent_resolver_ir_batcher.sendBatch" timed out after 1m0.047s (given timeout 59.99999946s): aborted during DistSender.Send: context deadline exceeded
        --- FAIL: TestTelemetry/multiregion/server (575.93s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTelemetry.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/telemetryccl/telemetryccl_test.TestTelemetry [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9761707?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9761707?buildTab=artifacts#/) on master @ [1f3419e178bdba544f74d9c9e14a4682efd18028](https://github.com/cockroachdb/cockroach/commits/1f3419e178bdba544f74d9c9e14a4682efd18028):


```
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.global
    telemetry.go:110: 
        testdata/telemetry/multiregion:193:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:197:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_table_in
    telemetry.go:110: 
        testdata/telemetry/multiregion:202:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2";
        ALTER TABLE t3 ADD COLUMN cr crdb_internal_region NOT NULL
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:207:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY ROW AS "cr"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_row_as
    telemetry.go:110: 
        testdata/telemetry/multiregion:212:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:216:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_table
    telemetry.go:110: 
        testdata/telemetry/multiregion:221:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:229:
         ALTER TABLE t4 SET LOCALITY REGIONAL BY ROW
        expected:
        sql.multiregion.alter_table.locality.from.global.to.regional_by_row
        
        found:
        error: pq: replica unavailable: (n1,s1):1 unable to serve request to r32:/NamespaceTable/{30-Max} [(n1,s1):1, (n2,s2):2, (n3,s3):3, next=4, gen=4]: closed timestamp: 1682317442.140790937,0 (2023-04-24 06:24:02); raft status: {"id":"1","term":8,"vote":"1","commit":267,"lead":"1","raftState":"StateLeader","applied":267,"progress":{"2":{"match":267,"next":291,"state":"StateReplicate"},"3":{"match":264,"next":291,"state":"StateReplicate"},"1":{"match":290,"next":291,"state":"StateReplicate"}},"leadtransferee":"0"}: have been waiting 62.50s for slow proposal RequestLease [/NamespaceTable/30,/Min)
        sql.multiregion.alter_table.locality.from.global.to.regional_by_row
        --- FAIL: TestTelemetry/multiregion/server (687.42s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTelemetry.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/telemetryccl/telemetryccl_test.TestTelemetry [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9783087?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9783087?buildTab=artifacts#/) on master @ [847de4595b62a0c03b14a3d0865cc8222fcf0f61](https://github.com/cockroachdb/cockroach/commits/847de4595b62a0c03b14a3d0865cc8222fcf0f61):


```
        ALTER TABLE t1 SET LOCALITY REGIONAL BY TABLE
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:179:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY ROW
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_row
    telemetry.go:110: 
        testdata/telemetry/multiregion:184:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:188:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY GLOBAL
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.global
    telemetry.go:110: 
        testdata/telemetry/multiregion:193:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:197:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_table_in
    telemetry.go:110: 
        testdata/telemetry/multiregion:202:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2";
        ALTER TABLE t3 ADD COLUMN cr crdb_internal_region NOT NULL
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:207:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY ROW AS "cr"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_row_as
    telemetry.go:110: 
        testdata/telemetry/multiregion:212:
         ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        expected:
        
        found:
        error: pq: polling for queued jobs to complete: poll-show-jobs: replica unavailable: (n2,s2):2 unable to serve request to r6:/Table/{0-4} [(n1,s1):1, (n2,s2):2, (n3,s3):3, next=4, gen=5]: closed timestamp: 1682408783.788100941,0 (2023-04-25 07:46:23); raft status: {"id":"2","term":14,"vote":"2","commit":281,"lead":"2","raftState":"StateLeader","applied":281,"progress":{"1":{"match":281,"next":301,"state":"StateReplicate"},"2":{"match":300,"next":301,"state":"StateReplicate"},"3":{"match":278,"next":301,"state":"StateReplicate"}},"leadtransferee":"0"}: have been waiting 60.50s for slow proposal RequestLease [/Table/0,/Min)
        --- FAIL: TestTelemetry/multiregion/server (953.62s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTelemetry.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/telemetryccl/telemetryccl_test.TestTelemetry [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9804966?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9804966?buildTab=artifacts#/) on master @ [2f4231084e27760b4927e15da9b6d46705dc8552](https://github.com/cockroachdb/cockroach/commits/2f4231084e27760b4927e15da9b6d46705dc8552):


```
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.global
    telemetry.go:110: 
        testdata/telemetry/multiregion:193:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:197:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_table_in
    telemetry.go:110: 
        testdata/telemetry/multiregion:202:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2";
        ALTER TABLE t3 ADD COLUMN cr crdb_internal_region NOT NULL
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:207:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY ROW AS "cr"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_row_as
    telemetry.go:110: 
        testdata/telemetry/multiregion:212:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:216:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_table
    telemetry.go:110: 
        testdata/telemetry/multiregion:221:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:229:
         ALTER TABLE t4 SET LOCALITY REGIONAL BY ROW
        expected:
        sql.multiregion.alter_table.locality.from.global.to.regional_by_row
        
        found:
        error: pq: replica unavailable: (n2,s2):2 unable to serve request to r32:/NamespaceTable/{30-Max} [(n1,s1):1, (n2,s2):2, (n3,s3):3, next=4, gen=4]: closed timestamp: 1682494228.654344067,0 (2023-04-26 07:30:28); raft status: {"id":"2","term":7,"vote":"2","commit":284,"lead":"2","raftState":"StateLeader","applied":284,"progress":{"1":{"match":284,"next":307,"state":"StateReplicate"},"2":{"match":305,"next":306,"state":"StateReplicate"},"3":{"match":273,"next":307,"state":"StateReplicate"}},"leadtransferee":"0"}: have been waiting 62.50s for slow proposal RequestLease [/NamespaceTable/30,/Min)
        sql.multiregion.alter_table.locality.from.global.to.regional_by_row
        --- FAIL: TestTelemetry/multiregion/server (767.31s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTelemetry.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/telemetryccl/telemetryccl_test.TestTelemetry [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9826992?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9826992?buildTab=artifacts#/) on master @ [5608a146973cb002756eef7bcafd002eb038afb4](https://github.com/cockroachdb/cockroach/commits/5608a146973cb002756eef7bcafd002eb038afb4):


```
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:197:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_table_in
    telemetry.go:110: 
        testdata/telemetry/multiregion:202:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2";
        ALTER TABLE t3 ADD COLUMN cr crdb_internal_region NOT NULL
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:207:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY ROW AS "cr"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_row_as
    telemetry.go:110: 
        testdata/telemetry/multiregion:212:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:216:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_table
    telemetry.go:110: 
        testdata/telemetry/multiregion:221:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    telemetry.go:110: 
        testdata/telemetry/multiregion:229:
        feature-usage [0 args]
        ALTER TABLE t4 SET LOCALITY REGIONAL BY ROW
        ----
        sql.multiregion.alter_table.locality.from.global.to.regional_by_row
    telemetry.go:110: 
        testdata/telemetry/multiregion:234:
         ALTER TABLE t4 SET LOCALITY GLOBAL
        expected:
        
        found:
        error: pq: polling for queued jobs to complete: poll-show-jobs: replica unavailable: (n2,s2):2 unable to serve request to r13:/Table/1{1-2} [(n1,s1):1, (n2,s2):2, (n3,s3):3, next=4, gen=4]: closed timestamp: 1682580192.462633356,0 (2023-04-27 07:23:12); raft status: {"id":"2","term":24,"vote":"2","commit":2119,"lead":"0","raftState":"StateCandidate","applied":2119,"progress":{},"leadtransferee":"0"}: have been waiting 60.50s for slow proposal HeartbeatTxn [/Table/11/2/"\x80"/131/1/2023-04-27T07:25:41.869026Z/1/0,/Min), [txn: c8954307]
        --- FAIL: TestTelemetry/multiregion/server (558.40s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTelemetry.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/telemetryccl/telemetryccl_test.TestTelemetry [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9850005?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9850005?buildTab=artifacts#/) on master @ [3b1bc872bb4bf8e5eb2ad8790ba8ec781167dd1f](https://github.com/cockroachdb/cockroach/commits/3b1bc872bb4bf8e5eb2ad8790ba8ec781167dd1f):


```
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    datadriven.go:143: 
        testdata/telemetry/multiregion:197:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_table_in
    datadriven.go:143: 
        testdata/telemetry/multiregion:202:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2";
        ALTER TABLE t3 ADD COLUMN cr crdb_internal_region NOT NULL
        ----
    datadriven.go:143: 
        testdata/telemetry/multiregion:207:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY ROW AS "cr"
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_row_as
    datadriven.go:143: 
        testdata/telemetry/multiregion:212:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    datadriven.go:143: 
        testdata/telemetry/multiregion:216:
        feature-usage [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE
        ----
        sql.multiregion.alter_table.locality.from.regional_by_table_in.to.regional_by_table
    datadriven.go:143: 
        testdata/telemetry/multiregion:221:
        exec [0 args]
        ALTER TABLE t3 SET LOCALITY REGIONAL BY TABLE IN "ap-southeast-2"
        ----
    datadriven.go:143: 
        testdata/telemetry/multiregion:229:
        feature-usage [0 args]
        ALTER TABLE t4 SET LOCALITY REGIONAL BY ROW
        ----
        sql.multiregion.alter_table.locality.from.global.to.regional_by_row
    datadriven.go:143: 
        testdata/telemetry/multiregion:234:
         ALTER TABLE t4 SET LOCALITY GLOBAL
        expected:
        
        found:
        error: pq: polling for queued jobs to complete: poll-show-jobs: replica unavailable: (n2,s2):3 unable to serve request to r13:/Table/1{1-2} [(n1,s1):1, (n3,s3):2, (n2,s2):3, next=4, gen=4]: closed timestamp: 1682671915.935326493,0 (2023-04-28 08:51:55); raft status: {"id":"3","term":40,"vote":"2","commit":2897,"lead":"0","raftState":"StateFollower","applied":2897,"progress":{},"leadtransferee":"0"}: have been waiting 63.00s for slow proposal RequestLease [/Table/11,/Min)
        --- FAIL: TestTelemetry/multiregion/server (1357.06s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTelemetry.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/telemetryccl/telemetry_test.go
+++ b/pkg/ccl/telemetryccl/telemetry_test.go
@@ -23,6 +23,7 @@ func TestTelemetry(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 	skip.UnderRace(t, "takes >1min under race")
+	skip.UnderDeadlock(t, "takes >1min under deadlock")
 
 	sqltestutils.TelemetryTest(
 		t,

--- a/pkg/sql/telemetry_test.go
+++ b/pkg/sql/telemetry_test.go
@@ -31,6 +31,7 @@ func TestTelemetry(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 	skip.UnderRace(t, "takes >1min under race")
+	skip.UnderDeadlock(t, "takes >1min under deadlock")
 
 	sqltestutils.TelemetryTest(
 		t,

